src/odemis/acq/align/autofocus.py

```diff
@@ -977,6 +1004,45 @@ def AutoFocusSpectrometer(spectrograph, focuser, detectors, selector=None, strea
     return f
 
 
+def CLSpotsAutoFocus(detector, focus, good_focus=None, rng_focus=None, method=MTD_EXHAUSTIVE):
```

Comment:
This is a complete method, which only changes the exposure time of the detector, and then does exactly as `AutoFocus()`. I'd suggest that the caller just does this. If in the future, more things needs to be done, then we could keep that function, but it might be simpler for now to just write:
```
detector.exposureTime.value = 0.01
return AutoFocus(...)
```